chatcommunicate.py

```diff
@@ -514,6 +514,12 @@ def get_message(id, host="stackexchange.com"):
 
 def dispatch_command(msg):
     command_parts = GlobalVars.parser.unescape(msg.content).split(" ", 1)
+    try:
+        if command_parts[0] == 'sh':
+            command_parts = command_parts[1].split(" ", 1)
+            command_parts[0] = "!!/" + command_parts[0]
```

Comment:
Why not just set `command_name` and `args` to `command_parts[0].lower()` and `command_parts[1]` respectively, and then have the logic for parsing `!!/` in an else statement? It would be functionally equivalent but a little easier to read IMO